runtime/coverage.go

```diff
@@ -245,6 +246,10 @@ func (r *CoverageReport) Percentage() string {
 // String returns a human-friendly message for the covered
 // statements percentage.
 func (r *CoverageReport) String() string {
+	percentage := r.Percentage()
```

Comment:
`Percentage` itself should probably check for `Statements() == 0` itself, and return "100%" in that case.

Reply:
@turbolent  Thank you for the comment. In a way 100% make sense - if there are no statements to cover then everything is covered. Although, if this method is used and combined with a JSON output like '{
  "coverage": {},
  "excluded_locations": [
    "I.Crypto",
    "I.Test",
    "s.7465737400000000000000000000000000000000000000000000000000000000"
  ]
}' or a similar summary this may be slightly misleading. And "NaN%" would still allow clients of Persentage() to treat it diffrently from 100%. @m-Peter told me that for example Percentage() would be used to generate a text summary sent to CI. But on the other hand pushing work on clients expecting a valid "string" number also isn't that good. I'll do as you proposed.